python/cuda/nvbench/__init__.pyi

```diff
@@ -0,0 +1,311 @@
+# Copyright 2025 NVIDIA Corporation
+#
+#  Licensed under the Apache License, Version 2.0 with the LLVM exception
+#  (the "License"); you may not use this file except in compliance with
+#  the License.
+#
+#  You may obtain a copy of the License at
+#
+#      http://llvm.org/foundation/relicensing/LICENSE.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# ============================================
+# PLEASE KEEP IN SYNC WITH py_nvbench.cpp FILE
+# ============================================
+# Please be sure to keep these type hints and docstring in sync
+# with the pybind11 bindings in ``../../src/py_nvbench.cpp``
+
+# Use mypy's stubgen to auto-generate stubs using
+# ``stubgen -m cuda.nvbench._nvbench`` and compare
+# stubs in generated out/cuda/nvbench/_nvbench.pyi
+# with definitions given here.
+
+from collections.abc import Callable, Sequence
+from typing import Optional, Self, SupportsFloat, SupportsInt, Union
+
+class CudaStream:
+    """Represents CUDA stream
+
+    Note
+    ----
+        The class is not directly constructible.
+    """
+    def __cuda_stream__(self) -> tuple[int]:
```

Comment:
question: in the docs below and in `py_nvbench.cpp` you return a pair. Should it be `tuple[int, int]`?